sass/themes/hsl/_theme.scss

```diff
@@ -14,6 +14,7 @@ $hsl-bus-express: #ca4000;
 $hsl-rail-red: #8c4799;
 $hsl-ferry-blue: #007a97;
 $hsl-blue-accessible: #0074bf;
+$hsl-external-gray: #c5cad2;
 
```

Comment:
I guess  this is not needed, jus define external-feeed-color directly.

Reply:
external-feed-color is now defined directly